lib/facter/facts/linux/ec2_metadata.rb

```diff
@@ -16,9 +16,13 @@ def call_the_resolver
       private
 
       def aws_hypervisors?
-        virtual = check_virt_what || check_xen || check_product_name || check_bios_vendor || check_lspci
+        virt = virtual
 
-        virtual == 'kvm' || virtual =~ /xen/
+        virt == 'kvm' || virt =~ /xen/ || virt == 'aws'
```

Comment:
I think that writing this way `virt.match?(/kvm|xen|aws/)` would be a bit more easy to understand

Reply:
Nice suggestion, changed!